src/elements/content-sidebar/Sidebar.js

```diff
@@ -82,13 +82,19 @@ class Sidebar extends React.Component<Props, State> {
 
     componentDidUpdate(prevProps: Props): void {
         const { fileId, history, isLarge, location }: Props = this.props;
-        const { fileId: prevFileId, isLarge: prevIsLarge }: Props = prevProps;
+        const { fileId: prevFileId, isLarge: prevIsLarge, location: prevLocation }: Props = prevProps;
         const { isDirty, isOpen }: State = this.state;
+        const { state = {} } = location;
```

Comment:
Maybe we should rename this as to not confuse it with the component state?

Reply:
Sure, no problem.